cxxopts.yaml

```diff
@@ -0,0 +1,48 @@
+# Generated from https://git.alpinelinux.org/aports/plain/main/cxxopts/APKBUILD
+package:
+  name: cxxopts
+  version: 3.2.1
+  epoch: 0
+  description: Lightweight C++ command line option parser as a header only library
+  copyright:
+    - license: MIT
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - cmake
+      - ninja
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/jarro2783/cxxopts.git
+      tag: v${{package.version}}
+      expected-commit: 4bf61f08697b110d9e3991864650a405b3dd515d
+      recurse-submodules: true
+
+  - runs: |
```

Comment:
We should be able to do this here instead:

```
- uses: cmake/configure

- uses: cmake/build

- uses: cmake/install
```

Reply:
This worked. melange convert suggests autoconf pipelines and I ran into issues there. I'll try to check out what the issue with that is.